packages/central-server/src/apiV2/import/importSurveys/Validator/ConfigValidator/EntityConfigValidator.js

```diff
@@ -117,6 +123,29 @@ export class EntityConfigValidator extends JsonFieldValidator {
     };
   }
 
+  constructPointsToMandatoryQuestion(rowIndex) {
+    return value => {
+      const questionCode = value;
+      const question = this.findOtherQuestion(questionCode, rowIndex, this.questions.length);
+
+      if (!question) {
+        throw new ValidationError(`Should reference another question in the survey`);
+      }
+
+      if (!question.validationCriteria) {
+        throw new ValidationError(`Referenced question should be mandatory`);
+      }
+
+      const { validationCriteria } = question;
+      const parsedValidationCriteria = convertCellToJson(validationCriteria);
+
+      if (!parsedValidationCriteria.mandatory || parsedValidationCriteria.mandatory !== 'true')
+        throw new ValidationError(`Referenced question should be mandatory`);
```

Comment:
```suggestion
      if (!parsedValidationCriteria.mandatory || parsedValidationCriteria.mandatory !== 'true') {
        throw new ValidationError(`Referenced question should be mandatory`);
       } 
```
Generally I don't mind if we include the curly brackets or not but it's nice to be consistent inside a function one way or the other